src/portal/lib/src/config/gc/gc.component.ts

```diff
@@ -97,6 +97,16 @@ export class GcComponent implements OnInit {
     this.disableGC = false;
   }
 
+  private resetSchedule(cron) {
+    this.schedule = {
+      schedule: {
+        type: this.CronScheduleComponent.scheduleType,
+        cron: cron
+      }
+    };
+    this.getJobs();
+  }
+
   getcron(cron: string) {
```

Comment:
why this method called getcron , actually it invoke a put operation to the backend?

Reply:
Yes, because it also gets cron from the cron component, now I think scheduleGc looks more appropriate and has been modified